site/docs/reference/Connectors/materialization-connectors/pinecone.md

```diff
@@ -25,30 +25,32 @@ To use this connector, you'll need:
 
 ## Embedding Input
 
-The materialization creates vector embeddings based on a text input from your collection. A field
-with type `string` must exist in your collection and it must be `required` in the collection schema.
+The materialization creates vector embeddings based on fields in your collection.
```

Comment:
this first sentence kinda makes it sound like each field gets its own embedding -- if I'm reading the rest correctly, in reality, each collection document results in one embedding. Assuming that's right, I suggest saying: "The materialization creates a vector embedding for each collection document. Its structure is based on the collection fields."